app/views/course/groups/_group_user_fields.html.slim

```diff
@@ -0,0 +1,4 @@
+= content_tag_for(:tr, f.object, class: 'nested-fields') do
```

Comment:
Yes, it's the partial for each user and follows the name convention of
cocoon.

On Thursday, June 18, 2015, Joel Low <notifications@github.com> wrote:

> In app/views/course/groups/_group_user_fields.html.slim
> <https://github.com/Coursemology/coursemology2/pull/336#discussion_r32691151>
> :
>
> > @@ -0,0 +1,4 @@
> > += content_tag_for(:tr, f.object, class: 'nested-fields') do
>
> Is this reusable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/336/files#r32691151>.
>